asati_ecr.tf

```diff
@@ -0,0 +1,66 @@
+# ArchiveSpace AirTable Integration (asati) containers
+# This is a standard ECR for an ECS with a Fargate launch type
+locals {
+  ecr_asati = "asati-${var.environment}"
+}
+module "ecr_asati" {
+  source            = "./modules/ecr"
+  repo_name         = "docker-asati"
+  login_policy_arn  = aws_iam_policy.login.arn
+  oidc_arn          = data.aws_ssm_parameter.oidc_arn.value
+  environment       = var.environment
+  tfoutput_ssm_path = var.tfoutput_ssm_path
+  tags = {
+    app-repo = "docker-asati"
```

Comment:
See the comment on line 8 above about naming. In particular, this line 14 needs to exactly match the name of the application repo (which I think Eric renamed to just `asati`).

Reply:
I've renamed the app repo.